pkg/apis/core/v1beta1/defaults.go

```diff
@@ -235,6 +236,15 @@ func SetDefaults_Shoot(obj *Shoot) {
 	if obj.Spec.Kubernetes.KubeAPIServer.EnableAnonymousAuthentication == nil {
 		obj.Spec.Kubernetes.KubeAPIServer.EnableAnonymousAuthentication = pointer.BoolPtr(false)
 	}
+
+	if k8sVersionGreaterOrEqualThan122 {
+		for i := 0; i < len(obj.Spec.Provider.Workers); i++ {
+			if obj.Spec.Provider.Workers[i].CRI != nil {
+				continue
+			}
+			obj.Spec.Provider.Workers[i].CRI = &CRI{Name: CRINameContainerD}
+		}
+	}
```

Comment:
This should be implemented also for the v1alpha1 version.

Reply:
done, thanks!